contrib/cmd/seccompagent/README.md

```diff
@@ -0,0 +1,57 @@
+# Seccomp Agent
+
+## Warning
+
+Please note this is an example agent, as such it is possible that specially
+crafted messages can produce bad behaviour. Please use it as an example only.
+
+Also, this agent is used for integration tests. Be aware that changing the
+behaviour can break the integration tests.
+
+## Get started
+
+Compile runc and seccompagent:
+```bash
+make all
+```
+
+Prepare a container:
+```bash
+mkdir container-seccomp-notify
+cd container-seccomp-notify
+mkdir rootfs
+docker export $(docker create busybox) | tar -C rootfs -xvf -
+```
+
+Run the seccomp agent in the background:
+```bash
+sudo ./contrib/cmd/seccompagent/seccompagent &
+```
+
+Copy the config.json in this directory to use for the runc container and start it:
```

Comment:
Not sure what you mean by "copy the config.json". A leftover from the older version?

Reply:
Ah, now I see. Maybe be more specific, like

```suggestion
Copy the example `config.json` file from the directory where this README.md is to the container
directory you prepared earlier (`container-seccomp-notify`), then start a container:```

Reply:
Added, thanks!